CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ Extended support to Image Streamer Rest API version 500 and 600 to the already e
    - Golden Image
    - OS Volume
 
+#### Bug fixes & Enhancements
```

Comment:
5.5.0 is already released. Can you please add a new release tag: 5.5.1 for bug fixes.

Reply:
Added the bug info under 5.5.1